lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.connection
+    @connection
```

Comment:
Confused by this connection method as in the Bookmarks walkthrough it said to take it out, but when I did that it started throwing errors even when I removed the test as well. Same for you?